node/utils.ts

```diff
@@ -52,3 +56,35 @@ export const hashString = (str: string) => {
 }
 
 export const getBucket = (prefix: string, bucketName: string) => `${prefix}_${bucketName}`
+
+export const startSitemapGeneration = async (ctx: Context) => {
+  const { clients: { vbase, events }, vtex: { logger, adminUserAuthToken } } = ctx
+  if (!adminUserAuthToken) {
+      ctx.status = 401
+      ctx.body = 'Missing adminUserAuth token'
+      logger.error(`Missing adminUserAuth token`)
+      return
+  }
```

Comment:
resolving the spacing. 

As you are mutating the value of `ctx.body`, you can only use the new value to log in case of an error.
 
```suggestion
  if (!adminUserAuthToken) {
    ctx.status = 401
    ctx.body = 'Missing adminUserAuth token'
    logger.error(ctx.body)
    return
  }
```

Reply:
I'm not sure if `logger.error` will be able to access the new value that was mutated, otherwise, the cool thing would be to put this in a const, so as not to duplicate code.